src/kudu/consensus/leader_election.cc

```diff
@@ -209,27 +209,31 @@ void FlexibleVoteCounter::FetchTopologyInfo() {
     }
   }
 
-  // Step 1: Compute number of voters in each region in the active config.
-  // As voter distribution provided in topology config can lag,
-  // we need to take into account the active voters as well due to
-  // membership changes.
-  AdjustVoterDistributionWithCurrentVoters(config_, &voter_distribution_);
-
-  // We assume that there are no voters in config_.peers() who
-  // are in present in regions not covered by voter_distribution_
-  // That is enforced via bootstrap and add-member
-  // The reverse is not true and has been handled in
-  // AdjustVoterDistributionWithCurrentVoters
+  if (adjust_voter_distribution_) {
```

Comment:
Leave a small comment that this handling is only for special cases like unsafe config change/remediate stuck leaders etc.